### ISSUE TITLE
geckodriver fails with: error: Unicode-aware Perl class not found (make sure the unicode-perl feature is enabled)

### ISSUE THREAD

**Issue by Whissi** [mozilla/geckodriver#1743]
On Gentoo Linux we are building Firefox from source. During that build we also enable geckodriver and let Firefox's build system build geckodriver next to the browser itself.

We just noticed that the resulting geckodriver isn't working, see the testcase below.

Any idea where to start?

## System

* Version: 0.26.0
* Platform: Gentoo Linux
* Firefox: 78.0.2

## Testcase

```
$ geckodriver &
$ curl -d '{"capabilities": {"alwaysMatch": {"acceptInsecureCerts": true}}}' http://localhost:4444/session
thread 'webdriver dispatcher' panicked at 'Error parsing version regexp: Syntax(
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
regex parse error:
    \d+\.\d+(?:[a-z]\d+)?
    ^^
error: Unicode-aware Perl class not found (make sure the unicode-perl feature is enabled)
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
)', testing/geckodriver/src/capabilities.rs:82:13
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
curl: (52) Empty reply from server
[1]+  Aborted                 geckodriver
```

**Comment by Whissi** [comment]
We found the problem! @gyakovlev was so kind to have a look and noticed that https://github.com/mozilla/gecko-dev/blob/master/testing/geckodriver/Cargo.toml#L24 has set
```
regex = { version="1.0", default-features = false, features = ["perf", "std"] }
```
Due to `default-features = false`, you disabled unicode feature which is required. The following patch works for us:
```diff
--- a/testing/geckodriver/Cargo.toml
+++ b/testing/geckodriver/Cargo.toml
@@ -21,7 +21,7 @@ mozdevice = { path = "../mozbase/rust/mozdevice" }
 mozprofile = { path = "../mozbase/rust/mozprofile" }
 mozrunner = { path = "../mozbase/rust/mozrunner" }
 mozversion = { path = "../mozbase/rust/mozversion" }
-regex = { version="1.0", default-features = false, features = ["perf", "std"] }
+regex = { version="1.3", default-features = false, features = ["perf", "std", "unicode"] }
 serde = "1.0"
 serde_derive = "1.0"
 serde_json = "1.0"
```
Note that I have bumped version requirement as well because version 1.0 has no unicode feature.

Can anybody help me getting this into firefox?

**Comment by whimboo** [comment]
Note that https://bugzilla.mozilla.org/show_bug.cgi?id=1649508 cares about that.

**Comment by whimboo** [comment]
Also we aren't going with the above mentioned Unicode approach. We removed that on purpose some releases ago to reduce the size of the geckodriver binary.

**Comment by jgraham** [comment]
This should be fixed in 0.27
